src/main/scala/com/yotpo/metorikku/metric/stepActions/dataQuality/ValidationRunner.scala

```diff
@@ -70,7 +103,7 @@ case class ValidationRunner() {
                 case "Column" => logByLevel(verificationResult.status, doubleMetricColumnConstrainFailedMsg.format((100 - (value * 100)), metric.name))
                 case "Dataset" => logByLevel(verificationResult.status, doubleMetricDataSetConstrainFailedMsg.format(value, metric.name))
                 case "Mutlicolumn" => logByLevel(verificationResult.status, doubleMetricColumnConstrainFailedMsg.format((100 - (value * 100)), metric.name))
-            }
+              }
```

Comment:
?

Reply:
Probably a bad auto-styling change that got away. I'll fix it.